app/Http/Controllers/Admin/RoomController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use App\Model\Room;
+use App\Model\Hotel;
+use App\Model\Image;
+use App\Http\Requests\Backend\RoomRequest;
+
+class RoomController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @param int $hotelId id of hotel
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index($hotelId)
+    {
+        Hotel::findOrFail($hotelId);
+        $columns = [
+            'id',
+            'name',
+            'descript',
+            'price',
+            'size',
+            'total',
+            'max_guest',
+            'hotel_id'
+        ];
+        $rooms = Room::select($columns)
+            ->with(['images'])
+            ->where('hotel_id', '=', $hotelId)
+            ->orderBy('id', 'DESC')
+            ->paginate(Room::ROW_LIMIT);
+        return view('backend.rooms.index', compact('rooms', 'hotelId'));
+    }
+
+        /**
+     * Show the form for creating a new room.
+     *
+     * @param int $hotelId id of hotel
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create($hotelId)
+    {
+        Hotel::findOrFail($hotelId);
```

Comment:
nếu muốn kiểm tra $hotelId tồn lại không, anh nghĩ e nên tạo 1 cái request rùi check exists in database